src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/confirmedarrival/ConfirmedArrival.kt

```diff
@@ -12,7 +12,7 @@ import javax.persistence.Table
 
 @Entity
 @Table(name = "confirmed_arrival")
-class ConfirmedArrival(
+data class ConfirmedArrival(
```

Comment:
`username` on here should be nullable - it probably hasn't mattered in the past as we only ever retrieve arrivals for today but we'll need that to cope with loading historic arrivals

Reply:
sorted